src/main/types.d.ts

```diff
@@ -32,7 +32,7 @@ declare interface Row {
 
 declare interface RowBasecampMapping {
   rowHash: string,
-  basecampTodoIds: string[]
+  roleTodoIdMap: { [key: string]: string }
```

Comment:
Maybe it's worth maybe a new type for this so we don't have `{ [key: string]: string }` floating around everywhere

Reply:
Yeah, I was getting bothered by that as well. Thanks for bringing it up. I created a simple type alias in the types file.